anyway/oauth.py

```diff
@@ -6,7 +6,7 @@
 from flask import current_app, url_for, request, redirect, Response
 from rauth import OAuth2Service
 
-from anyway.dataclasses.user_data import UserData
+from anyway.anyway_dataclasses.user_data import UserData
```

Comment:
I changed the folder name in my recent pull request - but this name is also good.